src/expr.cpp

```diff
@@ -7144,6 +7171,7 @@ static llvm::Value *lTypeConvAtomicOrUniformVector(FunctionEmitContext *ctx, llv
             }
             cast = exprVal;
             break;
+        // TODO!: all int casts are literally same, refactor to remove copy-paste
```

Comment:
There is a bunch of other places in this file like `ConstExpr` or `TypeCheckExpr` where we check particular atomic type. 
I believe that `AtomicType::TYPE_INT1` should be added to all these places as well.

Reply:
I have added more places with `TYPE_INT1` in the last commit.